imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -27,41 +33,51 @@ energy_attrs:
     VAR_TYPE: support_data
 
 epoch_attrs:
-  CATDESC: Time, number of nanoseconds since J2000 with leap seconds included
-  FIELDNAM: epoch
-  LABLAXIS: epoch
-  FILLVAL: -9223372036854775808
-  FORMAT: " " # Supposedly not required, fails in xarray_to_cdf
-  VALIDMIN: -9223372036854775808
-  VALIDMAX: 9223372036854775807
-  UNITS: ns
-  VAR_TYPE: support_data
-  SCALETYP: linear
-  MONOTON: INCREASE
-  TIME_BASE: J2000
-  TIME_SCALE: Terrestrial Time
-  REFERENCE_POSITION: Rotating Earth Geoid
+    <<: *default
+    CATDESC: Time, number of nanoseconds since J2000 with leap seconds included
+    FIELDNAM: epoch
+    LABLAXIS: epoch
+    FILLVAL: -9223372036854775808
+    FORMAT: " " # Supposedly not required, fails in xarray_to_cdf
+    VALIDMIN: -9223372036854775808
+    VALIDMAX: 9223372036854775807
+    UNITS: ns
+    VAR_TYPE: support_data
+    SCALETYP: linear
+    MONOTON: INCREASE
+    TIME_BASE: J2000
+    TIME_SCALE: Terrestrial Time
+    REFERENCE_POSITION: Rotating Earth Geoid
 
 # <=== Labels ===>
 energy_label:
     CATDESC: Energy per charge (E/q) sweeping step
+    DISPLAY_TYPE: ""
     FIELDNAM: Energy step
+    FILLVAL: ""
     FORMAT: A3
+    LABLAXIS: ""
+    REFERENCE_POSITION: ""
+    RESOLUTION: ""
+    TIME_BASE: ""
+    TIME_SCALE: ""
+    UNITS: ""
+    VALIDMAX: ""
+    VALIDMIN: ""
```

Comment:
I found that I needed to add these in order to pass the new check here: https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/a257b7e851f81fe0b468d726c584b08549c38943/imap_processing/cdf/cdf_attribute_manager.py#L309

Reply:
If I am understanding correctly, this is checking against whatever is defined as "required" in the `default_variable_cdf_attrs_schema.yaml` file. It loos like all of these are indeed required according to this file, for example `TIME_BASE`:

```yaml
attribute_key:
  # ===== EPOCH-ONLY VARIABLE ATTRIBUTES =====
  TIME_BASE:
    description: >
      fixed (0AD, 1900, 1970 (POSIX), J2000 (used by CDF_TIME_TT2000),
      4714 BC (Julian)) or flexible (provider-defined)
    required: true    # NOT Required in ISTP Guide
```

Reply:
Right. I am wondering if it should only be checking what is required and is present in the list of attributes defined for the datatype. Not important to this PR though.

Reply:
Based on slack convos, sounds like Maxine will fix this eventually. I will hang tight on merging this PR and update accordingly when the fix is in.